app/routes/index.js

```diff
@@ -59,7 +59,8 @@ export default Ember.Route.extend({
   },
   model() {
     return Ember.RSVP.hash({
-      member: this.store.findRecord('member', 4, { include: 'tags' }), // Get the member's record
+      member: this.store.findRecord('member', 4), //@TODO: Get the member's tags using findRecord 'include'
+      tags: this.store.query('tag', { memberId: 4, chapterId: 102 }), //@TODO: Get tags via the member's relationship
```

Comment:
This is actually more similar to how Solarium requests the data, so no need to add the `include` functionality yet.